### PR TITLE
Rails 4.2 back compat

### PR DESCRIPTION
**Pull request by andydust**
Support for Rails 4.2, backwards compatible with Rails 3.x. Add travis support and ability to test multiple versions of Rails.

### CODE REVIEW

**Comment by andydust** [comment]
meh, gonna change this to use appraisal

**Comment by andydust** [comment]
@lazyatom @csshsh think this is oke to merge?

File: lib/persist_campaign/controller.rb [hunk 1]
```diff
@@ -8,10 +8,22 @@ def self.included base
     module InstanceMethods
       # Solution provided by ndp's save-the-campaign gem (http://github.com/ndp/save-the-campaign),
       # reproduced here under the MIT license
-      def _compute_redirect_to_location_with_persist_campaign_params(options = {})
+
+      if Rails.version >= "4.2.1"
```
Comment (lazyatom) [inline lib/persist_campaign/controller.rb]: I _think_ this is probably safe, but I wonder if it's even safer to use `Gem::Version` instances here (http://stackoverflow.com/questions/2051229/how-to-compare-versions-in-ruby)

**Comment by gtd** [comment]
@csshsh is on vacation, so I'll step in and give it my :+1:.  We should also close out issues #1 and #2 when this is merged.

**Comment by andydust** [comment]
thanks.

**Pull request merged**